README.md

```diff
@@ -3,6 +3,30 @@
 
 kafka-connect-venafi-tpp is a [Kafka connector](http://kafka.apache.org/documentation.html#connect) for Venafi Trust Protection Platform security events.
 
+
+Description
+---
+This connector connects via HTTP to your instance of the Venafi Trust Protection Platform ( which shall be refered to TPP from here on) and pulls your Log events into kafka allowing you to do any filtering/transforming/processing you'd like to do within a comfortable kafka environment.  
+ **N.B.** Currently the connector starts from the of the beginning of time, a future release will allow the option of starting from now.
+
+EventLog object produced by this source **Connector** as defined by the Venafi WebSDK
+* **ClientTimestamp**: The time that the client generated the event.
+* **Component**: A string that identifies a component DN. For events that occur in a subsystem other than Config, such as a Secret Store, the Distinguished Name (DN) component may be blank.
+* **ComponentId**: The component ID that originated the event.
+* **ComponentSubsystem**: The component subsytem that originated the event.
+* **Grouping**: An integer for the component ID or Group Id that correlates to a set of events.
+* **Id**: A hexadecimal value that corresponds to an Event ID.
+* **Name**: The event name.
+* **ServerTimestamp**: The time the Trust Protection Platform server received the event.
+* **Severity**: A string value for one of these event severity: Emergency, Alert, Critical, Error, Warning, Notice, Info, or Debug
+* **SourceIP**: The IP address of the host that originated the event
+* **Text1**: A string variable, that contains all or part of an event message.
+* **Text2**: An additional string if present, that appears as part of an event message.
```

Comment:
```suggestion
* **Text2**: An additional string, that, if present, appears as part of an event message.
```
NB: I'm not a native speaker, so chances are my gut feeling here is 100% wrong... ;-)